weblate/trans/admin.py

```diff
@@ -84,8 +85,8 @@ class ProjectAdmin(WeblateModelAdmin, RepoAdminMixin):
 
     @admin.display(description=gettext_lazy("Administrators"))
     def list_admins(self, obj):
-        return ", ".join(
-            User.objects.all_admins(obj).values_list("username", flat=True)
+        return format_html_join_comma(
+            "{}", (User.objects.all_admins(obj).values_list("username", flat=False))
```

Comment:
`flat` defaults to false, make it simpler:

```suggestion
            "{}", User.objects.all_admins(obj).values_list("username")
```